app/util/exceptions.py

```diff
@@ -53,6 +53,10 @@ def __init__(self, message: str = "File too large to download or did not report
         """Thrown when we detect a file is dangerously large."""
         super().__init__(message, 413)
 
+class InvalidFileUrl(ISvcException):
+    def __init__(self, message: str = "The file URL was invalid. It is possible that the URL has expired"):
```

Comment:
Extreme nit: since the first sentence in the message uses punctuation, the second should as well.

```suggestion
    def __init__(self, message: str = "The file URL was invalid. It is possible that the URL has expired."):
```

Reply:
Haha I actually went back and forth on this